pyfar/classes/coordinates.py

```diff
@@ -1,9 +1,105 @@
-"""
-The following documents the pyfar coordinates class and functions for
-coordinate conversion. More background information is given in
-:py:mod:`coordinates concepts <pyfar._concepts.coordinates>`.
-Available sampling schemes are listed at
-:py:mod:`spharpy.samplings <spharpy.samplings>`.
+r"""
+The following introduces the
+:py:func:`Coordinates class <pyfar.classes.coordinates.Coordinates>`
+and the coordinate systems that are available in pyfar. Available sampling
+schemes are listed at :py:mod:`spharpy.samplings <spharpy.samplings>`.
+:ref:`Examples <gallery:/gallery/interactive/pyfar_coordinates.ipynb>` for
+working with Coordinates objects are part of the pyfar galler.
+
+Different coordinate systems are frequently used in acoustics research and
+handling sampling points and different systems can be cumbersome. The
+Coordinates class was designed with this in mind. It can convert between
+coordinate systems and rotate, query and plot coordinates points. Functions for
```

Comment:
```suggestion
handling sampling points and different systems can be cumbersome.
We designed a Coordinate class that internally stores coordinates in Cartesian coordinates and can convert them to the desired coordinate system on the fly by accessing and writing the data. 
Additionally the class can rotate, query and plot coordinates points. Functions for
```
this text sound a lot like the old implementation, maybe we can fix it here. I dont really like my suggestion,but nothing better comes into my mind.